diffkemp/cli.py

```diff
@@ -163,7 +178,8 @@ def make_argument_parser():
                         "control-flow-only",
                         "inverse-conditions",
                         "reordered-bin-ops",
-                        "group-vars"]
+                        "group-vars",
+                        "smt-sequential-blocks"]
```

Comment:
We don't support all kinds of operations within sequential blocks, do we? For instance memory reads/writes must exactly match. What about `sequential-alu-ops` for the name? I'm not sure if we need the `smt-` prefix either.